src/movies.js

```diff
@@ -1,14 +1,106 @@
+/////5/////
+const movies = [
+  {
+    title: 'The Shawshank Redemption',
+    year: 1994,
+    director: 'Frank Darabont',
+    duration: '2h 22min',
+    genre: ['Crime', 'Drama'],
+    score: 9.3
+  },
+  {
+    title: 'The Godfather',
+    year: 1972,
+    director: 'Francis Ford Coppola',
+    duration: '2h 55min',
+    genre: ['Crime', 'Drama'],
+    score: 9.2
+  },
+  {
+    title: 'The Godfather: Part II',
+    year: 1974,
+    director: 'Francis Ford Coppola',
+    duration: '3h 22min',
+    genre: ['Crime', 'Drama'],
+    score: 9
+  },
+  {
+    title: 'The Dark Knight',
+    year: 2008,
+    director: 'Christopher Nolan',
+    duration: '2h 32min',
+    genre: ['Action', 'Crime', 'Drama', 'Thriller'],
+    score: 9
+  },
+  {
+    title: '12 Angry Men',
+    year: 1957,
+    director: 'Sidney Lumet',
+    duration: '1h 36min',
+    genre: ['Crime', 'Drama'],
+    score: 8.9
+  },
+  {
+    title: 'Schindler"s List',
+    year: 1993,
+    director: 'Steven Spielberg',
+    duration: '3h 15min',
+    genre: ['Biography', 'Drama', 'History'],
+    score: 8.9
+  },
+  {
+    title: 'Pulp Fiction',
+    year: 1994,
+    director: 'Quentin Tarantino',
+    duration: '2h 34min',
+    genre: ['Crime', 'Drama'],
+    score: 8.9
+  }
+]
+
```

Comment:
If you would like to use this array for your testing, you can just require it: 

`const movies = require("./data")`